include/grpc++/impl/codegen/client_context.h

```diff
@@ -348,6 +348,13 @@ class ClientContext {
   /// Applications never need to call this method.
   grpc_call* c_call() { return call_; }
 
+  /// EXPERIMENTAL debugging API
+  ///
+  /// if status is not ok() for an RPC, this will return a detailed string
+  /// of the gRPC Core error that led to the failure. It should not be relied
+  /// upon for anything other than gaining more debug data in failure cases.
+  grpc::string debug_error_string() const { return debug_error_string_; }
```

Comment:
We may want to consider whether it makes sense to use a string_view as the return type :)

Reply:
Aren't we backing out of depending on abseil-cpp?

Reply:
Sorry I was talking about string_ref in https://github.com/grpc/grpc/blob/94e676e10f8c739289924b8458a246699e3623ce/include/grpc%2B%2B/impl/codegen/string_ref.h#L41
But I guess a string is fine too...